core/src/main/java/org/apache/metamodel/query/OperatorTypeImpl.java

```diff
@@ -75,40 +75,44 @@ public static OperatorType convertOperatorType(String sqlType) {
         if (sqlType != null) {
             sqlType = sqlType.trim().toUpperCase();
             switch (sqlType) {
-            case "=":
-            case "==":
-            case "EQ":
-            case "EQUALS_TO":
-                return OperatorType.EQUALS_TO;
-            case "<>":
-            case "!=":
-            case "NE":
-            case "NOT_EQUAL":
-            case "NOT_EQUAL_TO":
-            case "NOT_EQUALS":
-            case "NOT_EQUALS_TO":
-            case "DIFFERENT_FROM":
-                return OperatorType.DIFFERENT_FROM;
-            case ">":
-            case "GT":
-            case "GREATER_THAN":
-                return OperatorType.GREATER_THAN;
-            case ">=":
-            case "=>":
-            case "GREATER_THAN_OR_EQUAL":
-                return OperatorType.GREATER_THAN_OR_EQUAL;
-            case "IN":
-                return OperatorType.IN;
-            case "<":
-            case "LT":
-            case "LESS_THAN":
-                return OperatorType.LESS_THAN;
-            case "<=":
-            case "=<":
-            case "LESS_THAN_OR_EQUAL":
-                return OperatorType.LESS_THAN_OR_EQUAL;
-            case "LIKE":
-                return OperatorType.LIKE;
+                case "=":
+                case "==":
+                case "EQ":
+                case "EQUALS_TO":
+                    return OperatorType.EQUALS_TO;
+                case "<>":
+                case "!=":
+                case "NE":
+                case "NOT_EQUAL":
+                case "NOT_EQUAL_TO":
+                case "NOT_EQUALS":
+                case "NOT_EQUALS_TO":
+                case "DIFFERENT_FROM":
+                    return OperatorType.DIFFERENT_FROM;
+                case ">":
+                case "GT":
+                case "GREATER_THAN":
+                    return OperatorType.GREATER_THAN;
+                case ">=":
+                case "=>":
+                case "GREATER_THAN_OR_EQUAL":
+                    return OperatorType.GREATER_THAN_OR_EQUAL;
+                case "NOT IN":
+                    return OperatorType.NOT_IN;
+                case "IN":
+                    return OperatorType.IN;
+                case "<":
+                case "LT":
+                case "LESS_THAN":
+                    return OperatorType.LESS_THAN;
+                case "<=":
+                case "=<":
+                case "LESS_THAN_OR_EQUAL":
+                    return OperatorType.LESS_THAN_OR_EQUAL;
+                case "LIKE":
+                    return OperatorType.LIKE;
+                case "NOT_LIKE":
```

Comment:
We should definately add "NOT LIKE" here also (with space instead of underscore).